src/app/modules/rup/services/elementosRUP.service.ts

```diff
@@ -128,22 +134,33 @@ export class ElementosRUPService {
      *
      * @param {ISnomedConcept} concepto Concepto de SNOMED
      * @param {boolean} esSolicitud Indica si es una solicitud
+     * @param {string} origen Indica si es una solicitud
      * @returns {IElementoRUP} Elemento que implementa el concepto
      * @memberof ElementosRUPService
      */
-    buscarElemento(concepto: ISnomedConcept, esSolicitud: boolean): IElementoRUP {
+    buscarElemento(concepto: ISnomedConcept, esSolicitud: boolean, esIndicacion?: boolean): IElementoRUP {
         // Busca el elemento RUP que implemente el concepto
         if (typeof concepto.conceptId === 'undefined') {
             concepto = concepto[1];
         }
         concepto.semanticTag = concepto.semanticTag === 'plan' ? 'procedimiento' : concepto.semanticTag;
         if (esSolicitud) {
-            const elemento = this.cacheParaSolicitud[concepto.conceptId];
-            if (elemento) {
-                return this.populateElemento(elemento, esSolicitud);
+            if (esIndicacion) {
+                const elemento = this.cacheParaIndicaciones[concepto.conceptId];
+                if (elemento) {
+                    return this.populateElemento(elemento, esSolicitud);
+                } else {
+                    return this.populateElemento(this.defaultsParaSolicitud[concepto.semanticTag], esSolicitud);
+                }
             } else {
-                return this.populateElemento(this.defaultsParaSolicitud[concepto.semanticTag], esSolicitud);
+                const elemento = this.cacheParaSolicitud[concepto.conceptId];
+                if (elemento) {
+                    return this.populateElemento(elemento, esSolicitud);
+                } else {
+                    return this.populateElemento(this.defaultsParaSolicitud[concepto.semanticTag], esSolicitud);
+                }
             }
+
```

Comment:
Lo probé y anda bien 👍🏼 
Me pasa que este bloque me lo imagino asi, pero es solo una opción mas
```suggestion
const elemento = esIndicacion ? this.cacheParaIndicaciones[concepto.conceptId] : this.cacheParaSolicitud[concepto.conceptId];

            if (elemento) {
                return this.populateElemento(elemento, esSolicitud);
            } else {
                return this.populateElemento(this.defaultsParaSolicitud[concepto.semanticTag], esSolicitud);
            }
```